src/estimators/pose.cc

```diff
@@ -349,4 +358,161 @@ bool RefineRelativePose(const ceres::Solver::Options& options,
   return summary.IsSolutionUsable();
 }
 
+bool RefineGeneralizedAbsolutePose(
+    const AbsolutePoseRefinementOptions& options,
+    const std::vector<char>& inlier_mask,
+    const std::vector<Eigen::Vector2d>& points2D,
+    const std::vector<Eigen::Vector3d>& points3D,
+    const std::vector<size_t>& camera_idxs,
+    const std::vector<Eigen::Vector4d>& rig_qvecs,
+    const std::vector<Eigen::Vector3d>& rig_tvecs, Eigen::Vector4d* qvec,
+    Eigen::Vector3d* tvec, std::vector<Camera>* cameras,
+    Eigen::Matrix6d* qtvec_covariance) {
```

Comment:
Is the covariance computed in the tangent space, otherwise the name would imply that we get back a 7x7 matrix here?

Reply:
Yes this is in the tangent 3-DoF space, not in the ambient quaternion space - what would be a good unambiguous variable name? `rot_tvec_covariance` ? `axisangle_tvec_covariance` ?

Reply:
`rot_tvec_covariance` could be a good name, maybe with an explicit comment that refers to tangent space / angle-axis.

Reply:
Addresses.